llama_stack/providers/impls/braintrust/scoring/braintrust.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+from typing import List
+
+from llama_models.llama3.api.datatypes import *  # noqa: F403
+from llama_stack.apis.scoring import *  # noqa: F403
+from llama_stack.apis.scoring_functions import *  # noqa: F403
+from llama_stack.apis.common.type_system import *  # noqa: F403
+from llama_stack.apis.datasetio import *  # noqa: F403
+from llama_stack.apis.datasets import *  # noqa: F403
+from llama_stack.providers.datatypes import ScoringFunctionsProtocolPrivate
+
+from .config import BraintrustScoringConfig
+
+from .scoring_fn.braintrust_scoring_fn import BraintrustScoringFn
```

Comment:
I don't think we need this additional piece of indirection because I don't think you are going to have various scoring functions "inlined" as an impl. Should this be an adapter also and be called `remote::braintrust`?

Reply:
We have `BraintrustScoringFn(BaseScoringFn)` as a helper interface to manage supported function registry &  braintrust evaluators. 

I'm not quite sure about whether it should be remote/inline. We're not currently querying any of braintrust's API, we're using it as a library inline, and it is using OpenAI inference for judge.

Reply:
unwrapped BraintrustScoringFn into scoring impl!